packages/browser-wallet/package.json

```diff
@@ -33,6 +35,7 @@
         "react-virtualized-auto-sizer": "^1.0.6",
         "react-window": "^1.8.7",
         "react-window-infinite-loader": "^1.0.8",
+        "ts-jest": "^28.0.8",
```

Comment:
Move back to devDependencies and allow config.ts files to use devDependencies in eslinrc.js